jobs/innovint_contract_django.html

```diff
@@ -0,0 +1,34 @@
+---
+title: Django Backend Contract Project
+company: InnoVint
+url: www.innovint.us
+location: SF bay area & Portland (Remote OK!)
```

Comment:
It has to be one location for now - otherwise your job will not show up on the map. You can say that remote work is OK in the actual description.